### DISCUSSION TITLE
Please add a default aria attribute of `aria-current`

### DISCUSSION THREAD

**Discussion by iJuulia** [tailwindlabs/tailwindcss#14195] [Ideas]
This is used in react-router-dom's <NavLink> to mark the current page. I believe it would be very useful for it to be a default modifier.

**Comment by franktucker907** [comment]
Hello,
It sounds like you’re referring to the NavLink component in react-router-dom, which is often used to highlight the current page. By default, NavLink applies an activeClassName or activeStyle to the active link, making it easy to style the current page differently.
Making this a default modifier could indeed streamline the process and reduce the need for additional configuration. It would be a great suggestion to bring up with the maintainers of react-router-dom or even contribute to the project if you’re interested in open-source development!  
[Official Website](https://www-flyingtogether.com)
Best Regards
franktucker907

**Comment by steadygaze** [comment]
Same; Leptos router sets `aria-current="page"`[[1]](https://docs.rs/leptos_router/latest/leptos_router/components/fn.A.html) to indicate an anchor is pointing to the current page. I'd like to style the link to the current page differently in the navbar.

**Comment by jacobproffer** [comment]
This would be a great addition. Currently, I've been handling `aria-current` via: `aria-[current='page']` to do something such as `dark:aria-[current='page']:text-white`.
